### DISCUSSION TITLE
Bug found

### DISCUSSION THREAD

**Discussion by ghost** [community/community#59983] [Mobile] [answered]
On the mobile app, when you are redirected within the app to the profile page, the back button returns to the previous page but when you open from the home screen (where the navBar is visible) and the back button is pressed, the app crashes. There should not be a back button on the profile layout when entering directly from the navBar, only when redirected

**Comment by alcere** [comment]
Hi @Faimue! Thanks for the feedback! I share your opinion, I have created an internal issue and we will revisit the current behaviour.

**Reply by ghost** [reply]
Thanks, hopefully you can get this issue sorted out ASAP 😊

**Reply by alcere** [reply, marked as answer]
This issue has been fixed and it will included in Beta 1.121.0, scheduled to be released today.
